build.psm1

```diff
@@ -2094,7 +2094,7 @@ function script:Start-NativeExecution
     try {
         if($VerboseOutputOnError.IsPresent)
         {
-            $output = & $sb
+            $output = & $sb 2>&1
```

Comment:
It would be nice to have a comment here saying what this redirection does.

Reply:
2>&1 is a standard redirection in CMD, BASH and PowerShell.  I don't think we need to explain what all operator do in the code.